cmake/GzBenchmark.cmake

```diff
@@ -59,15 +59,6 @@ function(_gz_add_version_info_target)
   )
 endfunction()
 
-function(ign_add_benchmarks)
-  # TODO(chapulina) Enable warnings after all libraries have migrated.
```

Comment:
Note to self: we need to enable warnings in `gz-cmake3`.

Reply:
https://github.com/gazebosim/gz-cmake/pull/346